src/components/advanced-search/advanced-search-table.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <v-data-table
+      :headers="headers"
+      :items="[]"
+      :items-per-page="5"
+      class="elevation-1"
+      hide-default-footer
+    >
+      <template v-slot:item="row" class="advanced-search-results">
+        <tr>
+          <td></td>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+      </template>
+    </v-data-table>
+    <v-btn @click="mockSelectNR">Mock Select NR</v-btn>
```

Comment:
I agree for sure, the tab heights are the same besides the Retrieve NR tab. This height difference was called out explicitly in designs, as was the sliding transition :)
I have noticed this as well though, will see what i can do to smooth it out. (Without churning too much time 😁 )

Reply:
Yes, I remember the designs. Maybe Scott/Tracey will have an idea after seeing it in action.

I don't recall if the 2 tab sizes transition with vertical expand/shrink, or whether they use the left/right panning like the other tabs? Maybe they're using the left/right transition and it would look better with vertical expand/shrink?

Reply:
Still looking at ways to smooth this though. It's appears it is the buttons that make it look jumpy 🤔